src/http/one/ResponseParser.cc

```diff
@@ -179,8 +182,6 @@ Http::One::ResponseParser::parseResponseFirstLine()
         parsingStage_ = HTTP_PARSE_DONE;
         return 1; // no more parsing
     }
-
-    return 0; // need more to parse anything.
 }
```

Comment:
```suggestion

        // unreachable
        assert(false);
        return -1;
}
```

Or we can remove all those unnecessary `else` clauses. Your call.

Reply:
Done.